src/devices/tuya.ts

```diff
@@ -6543,6 +6543,54 @@ const definitions: Definition[] = [
             ],
         },
     },
+    {
+        fingerprint: [{modelID: 'TS0601', manufacturerName: '_TZE204_ugekduaj'}, {modelID: 'TS0601', manufacturerName: '_TZE200_ugekduaj'},
+            {modelID: 'TS0601', manufacturerName: '_TZE204_iwn0gpzz'}, {modelID: 'TS0601', manufacturerName: '_TZE200_iwn0gpzz'}],
+        model: 'SDM01',
+        vendor: 'TuYa',
+        description: 'Smart energy monitor for 3P+N system',
+        fromZigbee: [tuya.fz.datapoints],
+        toZigbee: [tuya.tz.datapoints],
+        configure: tuya.configureMagicPacket,
+        exposes: [tuya.exposes.voltageWithPhase('A'), tuya.exposes.voltageWithPhase('B'), tuya.exposes.voltageWithPhase('C'),
+            tuya.exposes.powerWithPhase('A'), tuya.exposes.powerWithPhase('B'), tuya.exposes.powerWithPhase('C'),
+            tuya.exposes.currentWithPhase('A'), tuya.exposes.currentWithPhase('B'), tuya.exposes.currentWithPhase('C'),
+            e.energy().withDescription('Total forward active energy'),
+            e.produced_energy().withDescription('Total reverse active energy'),
+            e.power_factor().withUnit('%').withDescription('Total power factor'), e.power().withDescription('Total active power'),
+            e.ac_frequency(),
+            tuya.exposes.energyWithPhase('A'), tuya.exposes.energyWithPhase('B'), tuya.exposes.energyWithPhase('C'),
+            tuya.exposes.energyProducedWithPhase('A'), tuya.exposes.energyProducedWithPhase('B'), tuya.exposes.energyProducedWithPhase('C'),
+            tuya.exposes.powerFactorWithPhase('A'), tuya.exposes.powerFactorWithPhase('B'), tuya.exposes.powerFactorWithPhase('C'),
+        ],
+        meta: {
+            tuyaDatapoints: [
+                [1, 'energy', tuya.valueConverter.divideBy100],
+                [2, 'produced_energy', tuya.valueConverter.divideBy100],
+                [15, 'power_factor', tuya.valueConverter.raw],
+                [101, 'ac_frequency', tuya.valueConverter.divideBy100],
+                [102, 'voltage_A', tuya.valueConverter.divideBy10],
+                [103, 'current_A', tuya.valueConverter.divideBy1000],
+                [104, 'power_A', tuya.valueConverter.raw],
+                [105, 'voltage_B', tuya.valueConverter.divideBy10],
+                [106, 'current_B', tuya.valueConverter.divideBy1000],
+                [107, 'power_B', tuya.valueConverter.raw],
+                [108, 'voltage_C', tuya.valueConverter.divideBy10],
+                [109, 'current_C', tuya.valueConverter.divideBy1000],
+                [110, 'power_C', tuya.valueConverter.raw],
+                [111, 'power', tuya.valueConverter.raw],
```

Comment:
Datapoints are the same as SPM02V2 but this one has more. I'm wondering if SPM02V2 also support the additional ones (like e.g. `[112, 'energy_A', tuya.valueConverter.divideBy100],`). Since you also added the SPM02V2 in https://github.com/Koenkk/zigbee-herdsman-converters/pull/6702, could you check this?

Reply:
Hello, I can confirm that SPM02V2 does not have these functions.
SDM01 is our new product, this is its unique function, I am the manufacturer of this product.